From af5c220050e90b388fdff4b3730cde150988daec Mon Sep 17 00:00:00 2001
From: Dahir Osman <dahir.osman@windriver.com>
Date: Fri, 23 Sep 2016 11:17:54 -0400
Subject: [PATCH] i40evf: Fix compile issue.

The Makefile was using the wrong CONFIG to compile the driver sources.
The driver was not being built at all without this fix.
---
 src/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Makefile b/src/Makefile
index b50a61d..610c5f9 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -28,7 +28,7 @@ ifneq ($(KERNELRELEASE),)
 # Makefile for the Intel(R) 10GbE PCI Express Virtual Function Driver
 #
 
-obj-$(CONFIG_IXGBE) += ixgbevf.o
+obj-$(CONFIG_IXGBEVF) += ixgbevf.o
 
 define ixgbevf-y
 	ixgbevf_main.o
-- 
1.9.1

